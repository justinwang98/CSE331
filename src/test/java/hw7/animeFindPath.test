# Load the anime dataset and search for an existing path.

LoadGraph anime anime.tsv

FindPath anime Sasuke Kirito
