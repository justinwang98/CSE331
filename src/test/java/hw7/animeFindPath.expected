# Load the anime dataset and search for an existing path.

loaded graph anime

path from Sasuke to Kirito:
Sasuke to Naruto with weight 1.000
Naruto to Kirito with weight 0.500
total cost: 1.500